### PR TITLE
Mx frontend

### PR DESCRIPTION
**Pull request by MXSH-Dev**
added user name, email, picture to profile page,
added link to explore page on drop down,
finished searching on both front and back, finished redirecting to course/instructor page from search or explore.

### CODE REVIEW

File: client/src/components/AsyncSearchBar.tsx [hunk 1]
```diff
@@ -1,57 +1,75 @@
...
   const [options, setOptions] = useState([] as any[]);
+  const history = useHistory();
 
-  const handleSearch = (query: string) => {
+  const handleSearch = (query: any) => {
     setIsLoading(true);
 
-    fetch(`${SEARCH_URI}?q=${query}+in:login&page=1&per_page=50`)
+    fetch(`http://127.0.0.1:5000/search/${query}`)
       .then((resp) => resp.json())
```
Comment (raysarinas) [inline client/src/components/AsyncSearchBar.tsx:17]: Should we use an Axios request here instead of `fetch()`? not sure if it matters

**Review by raysarinas — APPROVED** [review]
LGTM other than the one comment about `fetch`!

File: client/src/components/AsyncSearchBar.tsx (hunk 1 above)
Comment (MXSH-Dev) [reply client/src/components/AsyncSearchBar.tsx:17]: The offical doc of AsyncTypeahead lib uses fetch, I tried switch to axios, but keeping getting errors, I spend more than half hour trying to solve it but failed, so I just modified the example code using fetch.
Fetch and Axios are pretty much the same thing...

Comment (raysarinas) [reply client/src/components/AsyncSearchBar.tsx:17]: gotcha, just wanted to know if we could switch over just for consistency but all good if we need to use fetch 👍🏼

**Pull request merged**